bin/post_compile

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -eo pipefail # Exit at first error, including in a pipeline
+set -u # Consider unset variables as errors
+set -x # Print each command before executing it
+
+python manage.py tailwind build
+python manage.py collectstatic --noinput
```

Comment:
Can we switch this to a `pre_compile` script? Then we can drop running collect static twice as the build pack does this by default already

Reply:
`pre_compile` happens before dependencies are installed so we would need to write a script that installs them. I don't think that would be a better outcome.

And with this `post_compile`, we would not execute `collectstatic` twice: see my explanation at the bottom of the PR description.

Reply:
Right this makes sense now, apologies if I missed that explanation before. 

We should add that explanation in code somewhere. I will add it as an issue for now.